pyproject.toml

```diff
@@ -14,7 +14,7 @@ python = ">=3.9,<3.12"
 astrapy = "~0.6.2"
 cassio = "~0.1.3"
 unstructured = "^0.10"
-llama-index = { version = "0.9.14", extras = ["langchain"] }
+llama-index = { version = "0.9.16.post1", extras = ["langchain"] }
```

Comment:
Had to use `0.9.16.post1` and not `0.9.16` because llama-index uses `post` versions instead of increasing minor to fix bugs 😞 .